protocol/streaming/full_node_streaming_manager.go

```diff
@@ -490,19 +490,11 @@ func (sm *FullNodeStreamingManagerImpl) TracksSubaccountId(subaccountId satypes.
 	return exists
 }
 
-// SendOrderbookUpdates groups updates by their clob pair ids and
-// sends messages to the subscribers.
-func (sm *FullNodeStreamingManagerImpl) SendOrderbookUpdates(
```

Comment:
For the 3 functions `SendOrderbookUpdates`, `SendOrderbookFillUpdates`, `SendFinalizedSubaccountUpdates`, we are just taking out a few lines into a helper function. No logic change